cmd/flow-archive-client/main.go

```diff
@@ -95,15 +94,16 @@ func run() int {
 	}
 
 	// Decode the arguments
-	var args []cadence.Value
+	var args [][]byte
 	if flagParams != "" {
 		params := strings.Split(flagParams, ",")
 		for _, param := range params {
-			arg, err := convert.ParseCadenceArgument(param)
+			carg, err := convert.ParseCadenceArgument(param)
 			if err != nil {
 				log.Error().Err(err).Msg("invalid Cadence value")
 				return failure
 			}
+			arg, err := json.Encode(carg)
```

Comment:
```suggestion
			arg, _ := json.Encode(carg)
```

Reply:
not advocating for ignoring the error

Reply:
Yeah, I overlooked that, but it is in https://github.com/onflow/flow-archive/pull/136 that I will merge into this one.